### PR TITLE
Change unpacking of TF mobilebert inputs to use decorator

### PR DESCRIPTION
**Pull request by vumichien**
# What does this PR do?

As part of #16051 this PR changes the unpacking of the inputs of TF mobilebert to use the decorator instead.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

@Rocketknight1 @gante

### CODE REVIEW

File: src/transformers/models/mobilebert/modeling_tf_mobilebert.py [hunk 1]
```diff
@@ -936,6 +921,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutputWithPooling,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```
Comment (gante) [inline src/transformers/models/mobilebert/modeling_tf_mobilebert.py:924]: Sorry to be picky about this -- other models have been adding `@unpack_inputs` as the top decorator, can I ask you to move it to the top? That way we'll get consistent code :D

**Review by gante — COMMENTED** [review]
Thank you for tackling this <3 Two things:
1. Tests are failing due to formatting -- `make fixup` should fix it
2. Can you confirm that `RUN_SLOW=1 py.test -vv tests/[model_name]/test_modeling_tf_[model_name].py` runs?

File: src/transformers/models/mobilebert/modeling_tf_mobilebert.py (hunk 1 above)
Comment (vumichien) [reply src/transformers/models/mobilebert/modeling_tf_mobilebert.py]: Okay, I will move it

**Comment by vumichien** [comment]
@gante I have run the test and all pass. But when I run `make fixup` in my windows laptop it throwes the error `-n was unexpected at this time.
make: *** [Makefile:10: modified_only_fixup] Error 255
` Do you know how to fix it?

**Comment by gante** [comment]
> @gante I have run the test and all pass. But when I run `make fixup` in my windows laptop it throws the error `-n was unexpected at this time. make: *** [Makefile:10: modified_only_fixup] Error 255 ` Do you know how to fix it?

Check our guide [here](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#develop-on-windows) :)

**Review by gante — APPROVED** [review]
will merge as soon as CI turns green 👍

**Comment by vumichien** [comment]
Thank you :D

**Pull request merged**